test/functional/Valhalla/playlist.xml

```diff
@@ -25,13 +25,13 @@
 	<test>
 		<testCaseName>ValueTypeTests</testCaseName>
 		<variations>
-			<variation>NoOptions</variation>
-			<variation>-Xgcpolicy:nogc -XX:ValueTypeFlatteningThreshold=99999</variation>
-			<variation>-Xgcpolicy:optthruput -Xnocompactgc -XX:ValueTypeFlatteningThreshold=99999</variation>
+			<variation>-Xint</variation>
+			<variation>-Xint -Xgcpolicy:nogc -XX:ValueTypeFlatteningThreshold=99999</variation>
+			<variation>-Xint -Xgcpolicy:optthruput -Xnocompactgc -XX:ValueTypeFlatteningThreshold=99999</variation>
+			<variation>-Xjit:count=0</variation>
```

Comment:
We should also add `invocationCount = 2` and add an `-Xjit:count=1,disableAsyncCompilation` test as that is more realistic once we can inline methods.